src/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -233,27 +262,82 @@ def create(dataset, session_id, target, features=None, prediction_window=100,
 
     # Always create MXNet models, as the pred_model is later saved to the state
     # If MPS is used - the loss_model will be overwritten
-    loss_model, pred_model = _define_model_mxnet(len(target_map), prediction_window,
-                                                 predictions_in_chunk, context)
+    loss_model, pred_model = _define_model_mxnet(len(target_map), prediction_window, predictions_in_chunk, context)
 
     if use_mps:
         loss_model = _define_model_mps(batch_size, len(features), len(target_map),
                                        prediction_window, predictions_in_chunk, is_prediction_model=False)
 
         log = _fit_model_mps(loss_model, data_iter, valid_iter, max_iterations, verbose)
-
     else:
-        # Train the model using Mxnet
-        log = _fit_model_mxnet(loss_model, data_iter, valid_iter,
-                         max_iterations, num_mxnet_gpus, verbose)
+        from .._mxnet import mxnet as _mx
+        # Initialize the weights in MXNet and then pass them in TensorFlow
+        dummy_data_iter = _SFrameSequenceIter(chunked_data, len(features),
```

Comment:
Isn't this specific to `use_tensorflow`? Why are we doing this in the MXNet case too?

Could all this just be in `_fit_model_tf`?

Reply:
I did this to initialize the two with the same weights! I can move this since we are no longer comparing them with each other.